src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -4182,7 +4182,7 @@ void continuation_enter_cleanup(MacroAssembler* masm) {
 
   __ ld(t0, Address(sp, ContinuationEntry::parent_offset()));
   __ sd(t0, Address(xthread, JavaThread::cont_entry_offset()));
-  __ add(fp, sp, (int)ContinuationEntry::size());
+  __ add(fp, sp, (int)ContinuationEntry::size() + 2 * wordSize /* Adjust leave() afterward, which is different from AArch64's */);
```

Comment:
Maybe rephase like: "/* 2 extra words to match up with leave() */"
Also need to update comment at line 4168.